src/pause-play/js/main.js

```diff
@@ -124,11 +106,22 @@ class TestRunner {
     this.lastIterationTime;
   }
 
-  addPeerConnection() {
-    const videoElement = this.feedTable.addNewCell();
-    this.videoElements.push(videoElement);
+  addPeerConnection(elementType) {
+    const element = this.feedTable.addNewCell(elementType);
+    const constraints = {audio: true};
+    if (elementType === 'video') {
+      constraints.video = {
+        mandatory: {
+          maxWidth: 300,
+          minWidth: 300,
+        }
+      };
+    } else {
```

Comment:
I assume we will only support audio or video elements, add an else if (elementType === 'audio') clause here and an else that throws an error that the elementType provided is not supported.